boards/arm/lpcxpresso55s69/lpcxpresso55s69_ns.dts

```diff
@@ -79,3 +79,7 @@
 &hs_lspi {
 	status = "okay";
 };
+
+&wwdt0 {
```

Comment:
does it work on lpcxpresso55s69_cpu0 ? If yes, please add it to that.

Reply:
Thanks for suggestion, I've added relevant bits to lpcxpresso55s69_cpu0.dts and lpcxpresso55s69_cpu0.yaml files.